src/auth0-sso-login.js

```diff
@@ -170,7 +171,7 @@ export default class auth {
                 lock.getUserInfo(authResult.accessToken, (error, profile) => {
                   lock.hide();
                   if (error) {
-                    this.log(error);
+                    this.log('Error while retrieving user information after successful Auth0Lock authentication', error);
```

Comment:
This needs to be an object to match the current api. i.e. `this.log({ title: 'Title', error: error })`.

Actually should `ensureLogIn` fail when only the `getUserInfo` part fails?  I don't think so.  It will probably cause the wrong behavior to happen in our clients.  We should instead just not call the `profileRefreshed`.

Reply:
The current API of this library all logs just strings. `this.log` is a function, which clients of this library can subscribe to, and format their message to their needs.